output.go

```diff
@@ -100,8 +100,15 @@ func (t *Terminal) handleOutput(buf []byte) []byte {
 		size int
 		r    rune
 		i    = -1
+		cols = t.config.Columns
+		rows = t.config.Rows
 	)
 	for {
+		// check if terminal size has changed
+		if t.config.Columns != cols || t.config.Rows != rows {
+			break
```

Comment:
This makes me nervous... If we are resizing and there is stuff outputting it may not matter in apps like top, but if this is only being sent to the screen once won't this lead to it being lost?

Reply:
That's a good point.  I need to find a good test for that.

Reply:
Just ran it with a long apt update && apt upgrade and it was losing information.  Reverting change.